Allan-breakout-room/Bawan-Majid/styles.css

```diff
@@ -0,0 +1,193 @@
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+.logo {
+	width: 40px;
+	height: 40px;
+	display: inline-block;
+ margin-left: 20%;
+}
+.loc{width: 18px;
+height: 18px;
+display: inline-block;  
+}
+h5{
+	display: inline-block;
+	color:rgb(206, 202, 202) ;
+}
+.nav-box {
+	height: 40%;
+	width: 100%;
+	padding: 0;
+	margin: 0;
+	background-color: rgb(243, 240, 240);
+	position: relative;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.nav-bar {
+	display: inline-block;
+	position: absolute;
+	top: 15px;
+	right: 20%;
+}
+.overlay1{
```

Comment:
I would prefer to control the position form the parent itself and not style each child individually
check this code pen
https://codepen.io/mustafa7/pen/jObYKPK